less/index.less

```diff
@@ -53,7 +55,330 @@ table {
+	justify-content: space-between;
+}
+
+aside .author-image-container 	{
+	background-image: url("/images/about.jpg");
+	height: 150px;
+	width: 150px;
+	background-size: cover;
+	background-repeat: no-repeat;
+	background-position: center center;
+	border-radius: 50%;
+	margin: 0 auto;
+	position: relative;
+	margin-bottom: 30px;
+}
+
+aside .author-logo 	{
+	font-weight: 700;
+	font-size: 1.3rem;
+	width: 100%;
+	margin-bottom: .5em;
+}
+
+aside .author-logo 	{
+	display: block;
+	text-align: center;
+	color: #000;
+}
+
+aside .career 	{
+	display: block;
+	margin-bottom: 2em;
+	text-transform: uppercase;
+	font-size: .8rem;
+}
+
+aside nav 	{
+	width: 100%;
+	padding: 20px;
+	position: relative;
+}
+
+aside nav ul 	{
+	text-align: center;
+	margin: 0;
+	padding: 0;
+}
+
+aside nav ul li 	{
+	margin: 0 0 10px 0;
+	padding: 0;
+	text-transform: uppercase;
+	letter-spacing: 1px;
+}
+
+aside nav ul li  a 	{
+	display: block;
+	padding: 10px 0;
+	font-family: "Roboto Condensed";
+
+}
+
+aside footer 	{
+	text-align: center;
+	padding: 0 20px;
+	width: 100%;
+
+}
+
+aside footer ul 	{
+	padding: 0;
+	margin: 0;
+	list-style-type: none;
+}
+
+aside footer ul li 	{
+	display: inline;
+	margin: 0;
+	padding: 0;
+
+}
+
+aside footer ul li a 	{
+	padding: 5px
+}
+
+
+main 	{
+	width: calc(100% - 300px);
+	border: 1px solid blue;
+}
+
+.section-container 	{
+	padding-top: 50px;
+	padding-bottom: 50px;
+}
+
+.hero 	{
+	min-height: 500px;
+	border: 1px solid red;
+}
+
+.hero .hero-content-container 	{
+	display: flex;
+}
+
+.hero .hero-content-container .intro  	{
+	height: 450px;
+}
+
+.hero .hero-content-container .intro h1		{
+	margin-bottom: 20px;
+	font-size: 3rem;
+	line-height: 1.5;
+	font-weight: 700;
+}
+
+.hero .hero-content-container .intro h2 	{
+	font-size: 1.3rem;
+	line-height: 1.5;
+	margin-bottom: 30px;
+	font-weight: 300;;
+}
+
+.hero-image-container 	{
+	background: green;
+	// background-image: url("/images/img_bg_1.jpg");
+}
+
+.about-container, .portfolio-container, .contact-info-container 	{
+	padding: 0  2.5em;
+}
+
+.about-container .about-description, .portfolio-container .portfolio-header, .contact-header .contact-title 	{
+	font-size: 1.5rem;
+	margin-bottom: 3rem;
+	font-weight: 700;
+	text-transform: uppercase;
+	letter-spacing: 5px;
+	line-height: 1.8;
+	position: relative;
+}
+
+.portfolio-container .skills h2	{
+	font-size: 1.5rem;
+	padding: 10px;
+	
+}
+
+.portfolio-container .skills  ul 	{
+	padding: 0;
+	margin: 0;
+	list-style-type: none;
+}
+
+.portfolio-container .skills  ul li 	{
+	padding: 10px;
+}
+
+.portfolio-item-container   {
+	display: flex;
+	width: 100%;
+	justify-content: space-between;
+  }
+  
+  .portfolio-item-container  .portfolio-item  {
+	min-width: 48%;
+	margin-bottom: 20px;
+	text-align: center;
+  }
+
+
+  .portfolio-item img 	 	{
+	  display: block;
+	  width: 100%;
+  }
+
+  .portfolio-item h3 	{
+	  margin-top: 10px;
+	  padding: 10px 5px;
+	  font-size: 1.2rem;
+  }
+
```

Comment:
Try to add more comments to your code so it's more readable

Reply:
comment in css file or html file?